### PR TITLE
TEL-47 Adjust Linux package manager invocation for CentOS 7 and Amazon Linux 2

### PR DESCRIPTION
**Pull request by maxkondr**

### CODE REVIEW

File: metrics/package.go [hunk 1]
```diff
@@ -53,7 +54,7 @@ type Package struct {
 }
 
 // queryPkgFunc represents a function type for querying package information from particular package manager (dpkg or rpm).
-type queryPkgFunc func(ctx context.Context, packageName string) ([]*Package, error)
+type queryPkgFunc func(ctx context.Context, localOs, packageName string) ([]*Package, error)
```
Comment (artemgavrilov) [inline metrics/package.go:57]:
```suggestion
type queryPkgFunc func(ctx context.Context, localOS, packageName string) ([]*Package, error)
```

File: metrics/package_rhel.go [hunk 2]
```diff
@@ -4,42 +4,54 @@ import (
...
 	"fmt"
 	"os/exec"
 	"strings"
 
 	"go.uber.org/zap"
 )
 
-func queryRhelPackage(ctx context.Context, packageNamePattern string) ([]*Package, error) {
-	args, err := getRhelPackageManager()
+func queryRhelPackage(ctx context.Context, localOs, packageNamePattern string) ([]*Package, error) {
```
Comment (artemgavrilov) [inline metrics/package_rhel.go:14]:
```suggestion
func queryRhelPackage(ctx context.Context, localOS, packageNamePattern string) ([]*Package, error) {
```

**Review by artemgavrilov — APPROVED** [review]

File: metrics/package_rhel.go (hunk 2 above)
Comment (maxkondr) [reply metrics/package_rhel.go]: done

File: metrics/package.go (hunk 1 above)
Comment (maxkondr) [reply metrics/package.go]: done

**Pull request merged**